pkgs/os-specific/linux/cfs-zen-tweaks/default.nix

```
@@ -17,21 +17,16 @@ stdenv.mkDerivation rec {
     sha256 = "HRR2tdjNmWyrpbcMlihSdb/7g/tHma3YyXogQpRCVyo=";
   };
 
-  postPatch = ''
-    patchShebangs set-cfs-zen-tweaks.bash
-    chmod +x set-cfs-zen-tweaks.bash
-    substituteInPlace set-cfs-zen-tweaks.bash \
+  preConfigure = ''
+    substituteInPlace set-${pname}.bash \
```

```suggestion
    substituteInPlace set-cfs-zen-tweaks.bash \
```
substituting pname wherever possible just makes the code harder to read